Source/Tests/Engine/Engine.cpp

```diff
@@ -32,3 +34,11 @@ TEST_CASE("Engine started multiple times in same process")
         auto context = Tests::GetOrCreateContext(Tests::CreateCompleteContext);
     }
 }
+
+TEST_CASE("Random engine is stable for given seed")
+{
+    RandomEngine re(12);
+    REQUIRE(re.GetUInt() == 579251);
```

Comment:
do you really expect certain values from random generator?

Reply:
It's initialized with a seed, so seems like a solid test case for me.

Reply:
I have no idea what is behind this random, just thought that results can be different on different platforms

Reply:
The whole point of seeded random is to stay stable on every platform.
If it breaks, we have a problem.